packages/compass/src/main/window-manager.ts

```diff
@@ -127,6 +127,11 @@ function showConnectWindow(
       nodeIntegration: true,
       contextIsolation: false,
       enableRemoteModule: true,
+      nodeIntegrationInWorker: true,
+      // For local dev, electron can not load @mongosh/node-runtime-worker-thread
+      // worker (file:///) from the filesystem due to same-origin policy. For this
+      // reason we disable the webSecurity.
+      webSecurity: process.env.ENABLE_ELECTRON_WEB_SECURITY === '1',
```

Comment:
Can I suggest that we invert this flag, i.e. something like

```suggestion
      webSecurity: process.env.DISABLE_ELECTRON_WEB_SECURITY !== '1',
```

? Because that makes it harder to disable this setting purely by accident

Reply:
i spent quite sometime on deciding this tbh, but with your comment, i guess its easier other way! i'll make the change

Reply:
added in [d2851f9](https://github.com/mongodb-js/compass/pull/6203/commits/d2851f909a5198ceb309d35212a75fe262411439)